model/src/payout_curve.rs

```diff
@@ -76,8 +76,9 @@ impl Payouts {
         (leverage_long, leverage_short): (Leverage, Leverage),
         n_payouts: usize,
         fee: CompleteFee,
+        inverse_max_price_config: InverseMaxPrice,
```

Comment:
What do you think of not exposing this but creating a new function something like `new_inverse_deprecated` which defaults to `InverseMaxPrice::DoubleOfInitial`. That way we don't need to expose the constant outside of this module and the caller does not need to know about it.

Reply:
Yep, that's a better idea! Let me give it a go.

Reply:
I did it everywhere except for a few places in `model` itself. 

And I marked `InverseMaxPrice` and `new_inverse` as `pub(crate)` to ensure that they're not used elsewhere.